Pages/AlertPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Atata;
+
+namespace IFlow.Testing.Pages
+{
+    using _ = AlertPage;
+    [Url("https://demoqa.com/alerts")]
```

Comment:
To da sie zrobić trochę inaczej np
    using _ = AlertPage;
    [Url(AlertsUrl)]
    public class AlertPage: BasePage<_>
private const string AlertsUrl = "/alerts";


I base url jet znany i stały "demoqa" przenieść do base url w pliku atata.json dodać   "baseUrl": "https://demoqa.com",